models/drift_model.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"github.com/kamva/mgm/v3"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Drift struct {
+	mgm.DefaultModel `bson:",inline"`
+	Challenges       []primitive.ObjectID `bson:"challenges" json:"challenges" binding:"required"`
+	Prize            string               `bson:"prize" json:"prize" binding:"required"`
+	Points           int                  `bson:"points" json:"points" binding:"required"`
+	Group            primitive.ObjectID   `bson:"group" json:"group" binding:"required"`
```

Comment:
Could we just store an array of User IDs here instead of a Group? Groups are one-to-one with a drift so I don't know if we'll need a separate collection for them (a Group will only exist when there is a drift). 

One benefit of having a separate collection for Groups is if we want to implement "saved" Groups so that repeat drifters don't have to add their friends to the Group every time, but we should still have a conversation on how we want to structure this.

Reply:
We decided that your second point is legit. We resolved it.